static/templates/modals/dummy-confirm.html

```diff
@@ -0,0 +1,12 @@
+<div class="modal-header">
```

Comment:
We don't really, also wasn't too happy with mixing test files and real files. The reason for not testing against an existing template, is that the tests for a utility are using an unrelated piece of code, which breaks those tests' isolation a bit, and makes them dependent on an unrelated thing that may change or be removed at some point. How about we have a test templates dir for these (somehow didn't think of this earlier)?